app/assets/images/100/show.html.haml

```diff
@@ -0,0 +1,9 @@
+- if %w(middleware_datasources middleware_deployments).include?(@display)
```

Comment:
the location of this file is incorrect. the images folder contains only images.
most likely you meant something around `app/views`